docs/api-intro.md

```diff
@@ -0,0 +1,36 @@
+---
+id: api-intro
+title: aragonAPI Introduction
+sidebar_label: Introduction
+hide_title: true
+---
+
+![](/docs/assets/brand/aragonapi.png)
+
+Standard set of APIs and specifications used to interact with aragonOS-powered contracts by handling transaction pathing, upgradeability, and contract state without depending of a centralized service.
+
+## Reference implementations in specific languages
+
+### aragonJS
```

Comment:
```suggestion
### JavaScript
```

"aragonJS" isn't really a name we use. Could we instead change this section to be "JavaScript" so we can hide "aragon.js"?